metaflow/plugins/kfp/tests/sample_flows/nested_foreach_with_branching.py

```diff
@@ -0,0 +1,114 @@
+from metaflow import FlowSpec, step
+
+
+def truncate(var):
+    var = str(var)
+    if len(var) > 500:
+        var = "%s..." % var[:500]
+    return var
+
+
+class ExpectationFailed(Exception):
+    def __init__(self, expected, got):
+        super(ExpectationFailed, self).__init__(
+            "Expected result: %s, got %s" % (truncate(expected), truncate(got))
+        )
+
+
+def assert_equals(expected, got):
+    if expected != got:
+        raise ExpectationFailed(expected, got)
```

Comment:
why is this file using this `assert_equals`  and `ExpectationFailed` while the others aren't?
Consistency in code is of upmost importance.

Reply:
Could you update the other files to use this `assert_equals` helper function?

Reply:
You could create an issue for this in Github and create a separate PR to make it consistent. So we can merge and benefit from the work of this branch.  thank you @hsezhiyan !

Reply:
Sounds good, updating this in a separate PR:)